packages/unenv-preset/src/runtime/node/tls.ts

```diff
@@ -1,63 +0,0 @@
-import {
-	CLIENT_RENEG_LIMIT,
-	CLIENT_RENEG_WINDOW,
-	createSecurePair,
-	createServer,
-	DEFAULT_CIPHERS,
-	DEFAULT_ECDH_CURVE,
-	DEFAULT_MAX_VERSION,
-	DEFAULT_MIN_VERSION,
-	getCiphers,
-	rootCertificates,
-	Server,
-} from "unenv/node/tls";
-import type nodeTls from "node:tls";
-
-export {
-	CLIENT_RENEG_LIMIT,
-	CLIENT_RENEG_WINDOW,
-	createSecurePair,
```

Comment:
The PR to add this to workerd was closed without merging: https://github.com/cloudflare/workerd/pull/4690#issuecomment-3156479136

So I don't think we can just remove this file since that would be a breaking change to users that rely on that EOL function.